perfdash/config.go

```diff
@@ -384,10 +384,10 @@ func getProwConfig(configPaths []string) (Jobs, error) {
 	jobs := Jobs{}
 
 	for _, configPath := range configPaths {
-		fmt.Fprintf(os.Stderr, "Fetching config %s\n", configPath)
+		klog.Errorf("Fetching config %s\n", configPath)
```

Comment:
This does not seem like an error level (even though it was written to os.Stderr). It will be logged for every config path provided, so I'd keep it at INFO level, this is klog.Infof

Also - consider removing the newline char from all logging invocations - according to https://godoc.org/github.com/kubernetes/klog, klog appends newlines if they are not present.